## waypoints are of the form below
# waypoint_<two_characters>: {
#     x: <float>,
#     y: <float>,
#     theta: <float>
# }
waypoints:

    waypoint_TS: {
        x: 1.0,
        y: 1.0,
        theta: 0.0
    }
   
    waypoint_01: { #wpT
        x: 8.37,
        y: -6.26,
        theta: 0.0
    }

    waypoint_02: { #wpA
        x: 16.56,
        y: 5.61,
        theta: 0.0
    }

    waypoint_03: { #wpG
        x: 26.39,
        y: 4.35,
        theta: 0.0
    }

    waypoint_04: { #wpF
        x: 10.06,
        y: -14.79,
        theta: 0.0
    }

    landmark_01: {
        x: 9.80,
        y: 0.00,
        theta: 0.0
    }

    landmark_02: {
        x: 9.80, 
        y: 3.50, 
        theta: 0.0
    }

    landmark_03: {
        x: 34.00,
        y: 1.50,
        theta: 0.0
    }

    landmark_04: {
        x: 23.63,
        y: -4.62,
        theta: 0.0
    }

    landmark_05: {
        x: 10.27,
        y: 9.76,
        theta: 0.0
    }

    landmark_06: {
        x: 10.10,
        y: -21.38,
        theta: 0.0
    }

    landmark_07: {
        x: 5.44,
        y: -15.17,
        theta: 0.0
    }

    landmark_08: {
        x: 31.00,
        y: -9.13,
        theta: 0.0
    }

    landmark_09: {
        x: 18.37,
        y: 11.00,
        theta: 0.0
    }

    landmark_10: {
        x: 1.36,
        y: 9.60,
        theta: 0.0
    }

    landmark_11: {
        x: 17.00,
        y: -22.46,
        theta: 0.0
    }

    landmark_12: {
        x: 19.63,
        y: -0.02,
        theta: 0.0
    }

    landmark_13: {
        x: 18.29,
        y: -13.90,
        theta: 0.0
    }

    landmark_14: {
        x: 23.63,
        y: -4.62,
        theta: 0.0
    }

    landmark_15: { #prep_zone
        x: 3.02,
        y: -17.34,
        theta: -0.87
    }

    pass_point_01: {
        x: 0.68,
        y: -4.36, 
        theta: 1.70
    }

    pass_point_02: {
        x: 4.84, 
        y: -4.82, 
        theta: 1.53
    }

    pass_point_03: {
        x: 14.41, 
        y: -1.82, 
        theta: -2.01
    }

    pass_point_04: {
        x: -12.47, 
        y: 14.18, 
        theta: 1.28
    }

    pass_point_05: {
        x: 20.64, 
        y: -6.60, 
        theta: 0.02
    }

    pass_point_06: {
        x: 0.89,
        y: -9.37,
        theta: -2.50
    }

    pass_point_07: {
        x: -0.59,
        y: -2.54,
        theta: 0.0
    }

    pass_point_08: {
        x: 3.12,
        y: -4.63,
        theta: 1.72
    }

    pass_point_09: {
        x: 3.32,
        y: -4.66,
        theta: 0.0
    }

    pass_point_10: {
        x: 2.52,
        y: -7.53,
        theta: 0.0
    }

    pass_point_11: {
        x: 10.15,
        y: -4.28,
        theta: 2.02
    }

    pass_point_12: {
        x: 23.68,
        y: -0.74,
        theta: 0.53
    }

    pass_point_13: {
        x: 24.64,
        y: 2.17,
        theta: 3.10
    }

    pass_point_14: {
        x: 24.64,
        y: 2.17,
        theta: 0.0
    }

    pass_point_15: {
        x: 25.77,
        y: -7.13,
        theta: 1.39
    }

    pass_point_16: {
        x: -1.00,
        y: 16.07,
        theta: -1.39
    }

    pass_point_17: {
        x: 19.63,
        y: -10.19,
        theta: -0.06
    }

    pass_point_18: {
        x: -0.66,
        y: -6.08,
        theta: 3.0
    }






actions:
    ar_markers:
        ar_marker_1: ["waypoint_01"]
        ar_marker_3: ["waypoint_02"]
        ar_marker_10: ["waypoint_03"]
        ar_marker_5: ["waypoint_04"]
        ar_marker_8: ["waypoint_05"]


  

remote_path:
    start: "pass_point_07"
    pass_point_07: "pass_point_01"
    pass_point_01: "pass_point_08"
    pass_point_08: "pass_point_09" 
    pass_point_09: "pass_point_02"
    pass_point_02: "pass_point_10"
    pass_point_10: "waypoint_01" #wpT
    waypoint_01: "pass_point_11"
    pass_point_11: "pass_point_03"
    pass_point_03: "waypoint_02" #wpA
    waypoint_02: "landmark_12"    
    landmark_12: "pass_point_12"
    pass_point_12: "landmark_04"
    landmark_04: "pass_point_13"
    pass_point_13: "waypoint_03" #wpG
    waypoint_03: "pass_point_14"
    pass_point_14: "landmark_14"
    landmark_14: "pass_point_15"
    pass_point_15: "landmark_08"    
    landmark_08: "pass_point_16"
    pass_point_16: "pass_point_17"
    pass_point_17: "pass_point_05"
    pass_point_05: "landmark_13"
    landmark_13: "waypoint_04" #wpF
    waypoint_04: "landmark_07"
    landmark_07: "passpoint_06"
    pass_point_06: "pass_point_18"
    pass_point_18: "start"
    




